src/storage/main.cpp

```diff
@@ -487,13 +490,43 @@ static void signalHandler(sdeventplus::source::Signal& source,
     source.get_event().exit(EXIT_SUCCESS);
 }
 
+static void showUsage(char* appName)
+{
+    fprintf(stderr, R"(Usage: %s [options]
+Options:
+  -v, --verbose  Enable output debug messages.
+  -h, --help     Show this help
```

Comment:
So, where is the `--disable_debug_log` option you claim to add in the commit log?
Please also note that traditionally, the GNU long options use hyphens, not underscores (see for example https://www.gnu.org/prep/standards/standards.html#Option-Table), so please make it `--disable-debug-log`.